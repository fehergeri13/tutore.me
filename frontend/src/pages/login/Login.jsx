import React from 'react';


import "./login.less"
import Header from "../../components/header/Header";

export default class App extends React.Component {
    render() {
        return <div className="loginpage">
            <Header/>

            <div className="login">
                <h2>Bejelentkezés</h2>

                <form>
                    <label><span>Felhasználónév</span><input type="text" /></label>
                    <label><span>Jelszó</span><input type="password" /></label>
                    <label><span>Emlékezz rám</span><input type="checkbox"/></label>
                    <button type="submit">Belépés</button>
                </form>
            </div>

            <div className="register">
                <h2>Regisztráció</h2>

                <form>
                    <label><span>Felhasználónév</span><input type="text" /></label>
                    <label><span>Email</span><input type="text" /></label>
                    <label><span>Jelszó</span><input type="password" /></label>
                    <label><span>Jelszó megerősítése</span><input type="password" /></label>
                    <button type="submit">Regisztrálás</button>
                </form>
            </div>
        </div>;
    }
}
