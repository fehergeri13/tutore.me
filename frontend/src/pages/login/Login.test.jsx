import React from 'react';
import renderer from 'react-test-renderer';
import Login from "./Login";
import {Provider} from 'mobx-react';

describe('Login', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <Provider model={{}}><Login /></Provider>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
