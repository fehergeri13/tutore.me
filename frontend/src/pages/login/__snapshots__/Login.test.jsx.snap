// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<div
  className="login-page"
>
  <div
    className="login"
  >
    <h2>
      Bejelentkezés
    </h2>
    <form>
      <label>
        <span>
          Felhasználónév
        </span>
        <input
          type="text"
        />
      </label>
      <label>
        <span>
          Jelszó
        </span>
        <input
          type="password"
        />
      </label>
      <label>
        <span>
          Emlékezz rám
        </span>
        <input
          type="checkbox"
        />
      </label>
      <button
        type="submit"
      >
        Belépés
      </button>
    </form>
  </div>
  <div
    className="register"
  >
    <h2>
      Regisztráció
    </h2>
    <form>
      <label>
        <span>
          Felhasználónév
        </span>
        <input
          type="text"
        />
      </label>
      <label>
        <span>
          Email
        </span>
        <input
          type="text"
        />
      </label>
      <label>
        <span>
          Jelszó
        </span>
        <input
          type="password"
        />
      </label>
      <label>
        <span>
          Jelszó megerősítése
        </span>
        <input
          type="password"
        />
      </label>
      <button
        type="submit"
      >
        Regisztrálás
      </button>
    </form>
  </div>
</div>
`;
